Phytools removal

taddallas opened the pull request:
Attempting to quickly address the risk of dependency hell, starting with cutting out the need for `phytools` and `animation`, as referenced in issue #34 

I think `phytools` was only being used in the `plotBeta` function. Is this right? If not, then I need to hack away a bit more. I think it was only the `untangle` function though. 

Build was successful on Ubuntu 18.04 machine using `devtools::check()`

jarioksa commented:
Travis checks failed, but this seems to be independent of this change: building the test environment (R etc) failed before starting the test for this PR. I checked this with the latest R-devel ( 2019-12-12 r77564 ) with no problem. However, I think this feature was not checked in those standard tests.

My reading of this PR against full `phytools::untangle()` is positive: it takes the subset of the original code that we actually need here. Some small comments:

* We do not have to `importFrom ape Ntip` as it is not used in this code.
*  The PR unnecessarily changes 20 other files by re-formatting argument lists in their function definitions. While I think the new formatting is better (my editor does not like the over-long lines of R files in Hmsc), it confounds the essential change made here.
* Check indentation in the embedded `untangle`: it is confusing to human readers.

taddallas commented:
+ removed `Ntip` import. Good catch. 
+ I just ran `devtools::check()` to run the package through the standard checks and tests. I guess this messed with the formatting of the man files? What's the reason behind this?
+ I think I've fixed the indentation, but perhaps not. I'm used to each tab being 2 spaces, and I preferentially use tabs over spaces, which may translate **not well** across text editors. Also, the three space indent takes some getting used to (but hopefully I've done it correctly now).
